db/attachment.go

```diff
@@ -130,14 +143,15 @@ func (db *DatabaseCollectionWithUser) storeAttachments(ctx context.Context, doc
 			}
 		}
 	}
-	return newAttachmentData, nil
+
+	return newAttachments, nil
 }
 
-// retrieveV2AttachmentKeys returns the list of V2 attachment keys from the attachment metadata that can be used for
+// retrieveV2AttachmentIDs returns the list of V2 attachment keys from the attachment metadata that can be used for
```

Comment:
What does IDs represent here?   Is it meant to be name, or Attachment document key?